seed/seed.json

```diff
@@ -139,7 +139,7 @@
             "name": "DoNotShowCustomAdNotificationsOnBetaAndReleaseChannelsStudy",
```

Comment:
as this is not tagged to the confirmations DTO and the default for `should_show_custom_notifications` is `false`, should we maybe remove this study to keep things simple? https://github.com/brave/brave-core/blob/24fe16e3e30a7e75cf81f16bcbaa2d2982af17f0/components/brave_ads/browser/features.cc#L99

Reply:
These studies have `max_version` and are for backwards compatibility